api/Document.json

```diff
@@ -927,9 +927,15 @@
               },
               "chrome_android": "mirror",
               "edge": "mirror",
-              "firefox": {
-                "version_added": "131"
-              },
+              "firefox": [
+                {
+                  "version_added": "preview"
```

Comment:
> The option is now behind a pref

Shouldn't we add `version_added: "133"` mentioning the pref (`dom.shadowdom.new_caretPositionFromPoint_behavior.enabled`)?

Reply:
Once something is in preview the pref is considered irrelevant. I don't make those rules :-)